scripts/node_builder_glibc_env.sh

```diff
@@ -4,20 +4,29 @@
 # Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
 #
 set -eu
-# TODO support aarch64
-if [ "$(uname -m)" = "aarch64" ]; then
-    exit 0
-fi
 
-yum update -y && yum install -y curl python3 make git ca-certificates npm
+# x86_64 container: Centos7 -> python3.7 built by code from node_builder_glibc_env.sh
+# x86_64 exe: Centos7 -> python3.7 built by code from node_builder_glibc_env.sh
+# aarch64 container: Centos8 -> python3.6.8 (default python from centos8 image) -> building with --static-libstdc++
+# aarch64 exe: Centos8 -> python3.10 based from build-prepare stage where it is built -> building with --static-libstc++
+
+yum update -y && yum install -y curl make git ca-certificates npm
 # yum has node v10 by default, so we need to add newer version to run node-gyp
 curl -fsSL https://rpm.nodesource.com/setup_16.x | bash -
+yum remove -y nodejs npm
```

Comment:
Why is it needed? Especially due tonext line re-installing `nodejs`?

Reply:
There was some mismatch between nodejs version and npm version, but I guess we don't have to install it separately as nodejs package includes npm. I deleted this line